packages/qwik/src/cli/check-client/constant.ts

```diff
@@ -0,0 +1,6 @@
+import path from 'path';
+
+export const DISK_DIR: string = path.resolve('dist');
+export const SRC_DIR: string = path.resolve('src');
+export const BUILD_ARGS: string[] = ['run', 'build.client'];
```

Comment:
this should be in the run command itself, you're making it harder to know what's going on.